clang/test/dpct/cmake_migration/case_048/out/dpct.cmake

```diff
@@ -0,0 +1,132 @@
+#==---- dpct.cmake --------------------------------- cmake script file ----==//
```

Comment:
This change is not related to the current PR; pls submit a separate PR for this change.